client/startup/notifications/konchatNotifications.ts

```diff
@@ -98,6 +98,10 @@ Meteor.startup(() => {
 				'muteFocusedConversations',
 			);
 
+			if (notification.payload.type === 'l') {
```

Comment:
it doesnt look the best match only testing the 'l' to be sure its an queue notification